implementations/rust/ockam/ockam_command/tests/cmd_project.rs

```diff
@@ -33,8 +33,7 @@ fn valid_arguments() -> Result<(), Box<dyn std::error::Error>> {
     let mut cmd = Command::cargo_bin("ockam")?;
     cmd.args(prefix_args)
         .arg("authenticate")
-        .arg("--token")
-        .arg("02043d7bc316467b25b8df7118f4d1ba4b1911284236a3f94d8017ac7faff625");
+        .arg("7b226f6e655f74696d655f636f6465223a2263306633656163373862383238633665346564316263393935626261626431653432373436653332343137373332373564303230313165326137326263623133222c2270726f6a656374223a7b226e6f64655f726f757465223a222f646e73616464722f6b38732d6875626465762d6e67696e78696e672d383536373330343661622d363630656539343138303064346131312e656c622e75732d776573742d312e616d617a6f6e6177732e636f6d2f7463702f343031312f736572766963652f617069222c226964223a2232646163363536342d656537382d346261622d623938322d643432306238646261643966222c226e616d65223a2264656661756c74222c226964656e746974795f6964223a225030373461353735633839376163363534333431376235373935663836326434373836383932386639396634386438326165613661643230303637396532326131222c22617574686f72697479223a7b226964223a225035626265373235633934346133336439343566376530643363663235633036333066393232653030356563373538623931653063633838653565633030636131222c2261646472657373223a222f646e73616464722f6b38732d6875626465762d6e67696e78696e672d383536373330343661622d363630656539343138303064346131312e656c622e75732d776573742d312e616d617a6f6e6177732e636f6d2f7463702f343031322f736572766963652f617069222c226964656e74697479223a5b312c3136302c31362c3139352c37352c3137332c3231362c322c39392c3131322c3232312c32382c33382c312c3133312c3234332c32322c3133372c3132382c3139362c3136302c3231332c362c37352c3233302c3131342c3233382c3134362c34362c35372c3232322c3135332c3231362c302c352c37312c3230312c35302c35372c3138362c36312c3132392c3134322c3139342c3130382c3135362c3231382c3232312c34322c35332c3230332c3232332c33312c3136332c3138322c3230392c3136372c34392c3232342c39372c3130302c3137372c372c3135392c3138332c3138342c382c37392c36372c37352c36352c37372c39352c38322c37352c332c312c33322c302c302c302c33322c3131342c3232302c3139392c32312c38392c3232332c392c3134382c32382c352c36392c31342c32322c3235302c38332c38342c36312c3234392c31392c3130342c302c3134302c3135322c32372c3131312c3132312c36362c39312c39362c35332c3133322c3234352c332c312c312c36342c372c3233342c3138392c38302c392c35342c302c3132302c3137352c3136302c3233302c3138312c3133372c37392c3132312c3133362c3131392c3230352c352c31372c3138322c3139312c3134352c3130302c3235322c3231332c3133312c37372c3230302c3234342c32372c3134372c382c34332c3132312c35382c38382c31392c3131342c3139322c34342c3137322c362c37312c382c3134342c35312c3235312c3230352c3234392c39312c35302c3231362c3131332c3139312c31362c31302c3130352c36362c33362c3135352c3133362c332c31355d7d2c226f6b7461223a6e756c6c7d2c2274727573745f636f6e74657874223a6e756c6c7d");
```

Comment:
Is this maybe something we should grab via `include_str!` or similar? Without wrapping it's probably pretty harmless.